sdk/storage/src/core/clients/storage_account_client.rs

```diff
@@ -100,26 +103,50 @@ impl StorageAccountClient {
         http_client: Arc<Box<dyn HttpClient>>,
         blob_storage_url: &Url,
         table_storage_url: &Url,
+        queue_storage_url: &Url,
+        filesystem_url: &Url,
     ) -> Arc<Self> {
+        Self::new_emulator_with_account(
+            http_client,
+            blob_storage_url,
+            table_storage_url,
+            queue_storage_url,
+            filesystem_url,
+            "devstoreaccount1",
+            "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
```

Comment:
This will get flagged, assuming this is an auth token, even if ephemeral or fake. Cognitive Search even have a public, *read-only* sample account that we shared creds to - on docs.microsoft.com even - and was flagged. We do have a system in place for azure-sdk-for-net and some other languages to deal with this, but we need to get the pipeline fully built up and used. Perhaps, for now, this will be okay - as long as it is not actual valid creds into our Azure sub (if so, it needs to be rotated immediately) - but I would add a TODO comment or even BUGBUG we can scan to find these - or open a tracking issue even.

Reply:
I opened https://github.com/Azure/azure-sdk-for-rust/issues/280.